Dockerfile

```diff
@@ -10,9 +10,11 @@ ENV SONARQUBE_PLUGINS_DIR=/opt/sonarqube/default/extensions/plugins \
     SONARQUBE_JMX_ENABLED=false \
     DOCKERIZE_VERSION=v0.5.0
 
+USER root
```

Comment:
Do we need to explicitly set the user as root at the start?

Reply:
Yes, the base image has changed the user to sonarqube and Dockerize did not work without setting user to root.